glue/core/roi.py

```diff
@@ -163,21 +177,37 @@ class RectangularROI(Roi):
 
     """
     A 2D rectangular region of interest.
+
+    Parameters
+    ----------
+    xmin, xmax :  float, optional
+        x coordinates of left and right border
+    ymin, ymax :  float, optional
+        y coordinates of lower and upper border
+    theta : float, optional
+        Angle of anticlockwise rotation around center
     """
 
-    def __init__(self, xmin=None, xmax=None, ymin=None, ymax=None):
+    def __init__(self, xmin=None, xmax=None, ymin=None, ymax=None, theta=None):
         super(RectangularROI, self).__init__()
         self.xmin = xmin
         self.xmax = xmax
         self.ymin = ymin
         self.ymax = ymax
+        self.theta = 0 if theta is None else theta
+        if np.isclose(self.theta % np.pi, 0.0, atol=1e-9):
+            self.rotation = np.identity(2)
+        else:
+            self.rotation = rotation(self.theta)
+        self.invrot = self.rotation * [[1, -1], [-1, 1]]
```

Comment:
Maybe ``inv_rotation_matrix`` and make private as just a convenience? Though this makes it sound like it's the inverse of the matrix when it's actually the rotation in the opposite direction. I think we might want to default to using the anticlockwise rotation in the first place - see my top-level review comment.

Reply:
Yes, reviewing how often they are used it's probably not worth caching them at all, certainly not both of them.

Reply:
Removed as properties